miscmodule/showroll_command.go

```diff
@@ -0,0 +1,126 @@
+package miscmodule
+
+import (
+	"math"
+	"math/rand"
+	"regexp"
+	"strconv"
+	"strings"
+
+	bot "../sweetiebot"
+	"github.com/blackhole12/discordgo"
+)
+
+var diceregex = regexp.MustCompile("[0-9]*d[0-9]+")
+
+type showrollCommand struct {
+}
+
+func (c *showrollCommand) Info() *bot.CommandInfo {
+	return &bot.CommandInfo{
+		Name:              "Showroll",
```

Comment:
This should really be ShowRoll if we're going to bother with proper capitalization.